core/terraform/network/dns_zones.tf

```diff
@@ -95,6 +96,23 @@ resource "azurerm_private_dns_zone_virtual_network_link" "blobcore" {
   lifecycle { ignore_changes = [tags] }
 }
 
+resource "azurerm_private_dns_zone" "dfscore" {
+  name                = "privatelink.dfs.core.windows.net"
+  resource_group_name = var.resource_group_name
+  tags                = local.tre_core_tags
+  lifecycle { ignore_changes = [tags] }
+}
```

Comment:
This is already provisioned elsewhere (search for the name)

Reply:
removed